datalake-api/src/main/java/com/sequenceiq/sdx/api/model/SdxUpgradeRequest.java

```diff
@@ -33,6 +33,9 @@ public class SdxUpgradeRequest {
     @ApiModelProperty(ModelDescriptions.SKIP_DATAHUB_VALIDATION)
     private Boolean skipDataHubValidation;
 
+    @ApiModelProperty(ModelDescriptions.ROLLING_UPGRADE_ENABLED)
+    private Boolean rollingUpgradeEnabled;
```

Comment:
rollingUpgradeEnabled or rollingUpgradeRequested?

Reply:
We started to use `rollingUpgradeEnabled` in the StackV4 API so I think it's more consistent if we use `rollingUpgradeEnabled` in this API as well.